packages/pagination/src/FirstPage.tsx

```diff
@@ -42,16 +42,16 @@ const FirstPage = React.forwardRef<
   });
 
   return (
-    <Button
-      pill
-      ref={ref}
+    <a
+      ref={ref as Ref<HTMLAnchorElement>}
       {...props}
       aria-label={ariaLabel}
-      className={className}
+      className={classNames(className, ccPagination.button, ccPagination.icon)}
       rel="start"
+      role="button"
```

Comment:
Should this really have button as role? 🤔  It is navigation... right?

Reply:
Actually, you might be right that this should be announced as "link" instead of "button" by assistive technologies. I'll check that.

Reply:
Not sure if it is enclosed in a nav element? But there are things like this one: https://www.w3.org/TR/dpub-aria-1.0/#doc-pagelist ? 🤷 ...I'll leave it to you or the a11y team to decide. 😄

Reply:
Yes, It's enclosed in a <nav>. You don't need role="button" since these anchor tags are inside a <nav> but just an aria tag indicating the current page `aria-current="page"`